elytron/WFCORE-4360-CredentialStore_Expression_Resolution.adoc

```diff
@@ -0,0 +1,105 @@
+= WFCORE-4360 CredentialStore Backed Expression Resolution
+:author:            Darran Lofthouse
+:email:             darran.lofthouse@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The present `CredentialStore` integration is implemented to provide a tight integration between the locations in the model which require access to credentials, however their is also a desire to encrypt other attribute values stored in the management model.  This enhancement is specifically to focus on that latter requirement.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFCORE-4360[WFCORE-4360]
+* https://issues.jboss.org/browse/WFLY-12215[WFLY-12215]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-677[EAP7-677]
+* https://issues.jboss.org/browse/WFLY-11101[WFLY-11101]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:jkasik@redhat.com[Jan Kasik]
+
+=== Testing By
+
+[ ] Engineering
+[x] QE
+
+=== Affected Projects or Components
+
+Utility classes may be required within the WildFly Elytron project and will be developed as the need is identified, however the primary integration will be performed within the WildFly Core project as this enhancment is predominantly in relation to management model expression resolution.
+
+Additionally all documentation lives within the WildFly project so updates will be applied there accordingly.
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+
+The general intent for this enhancement is to develop some form of configurable expression resolver.
```

Comment:
Will be this expression resolver presented by a management resource? If it will can you please present an example?

Reply:
As I say in my previous comment I just need to verify the feasibility of this resolver and then I can start to add some anticipated configuration.